net/node/message.go

```diff
@@ -202,6 +203,7 @@ func (localNode *LocalNode) remoteMessageRouted(remoteMessage *nnetnode.RemoteMe
 				log.Errorf("Marshal new relay msg body error: %v", err)
 				return nil, nil, nil, false
 			}
+			atomic.AddUint64(&NodeRelayMessage, 1)
```

Comment:
As suggested by the Go doc: https://golang.org/pkg/sync/atomic/ it's better not to use sync/atomic directly. If you make it a LocalNode field, you can use the existing sync.RWMutex in LocalNode to handle atomic read/write.